mixer/pkg/adapter/test/integration.go

```diff
@@ -167,7 +172,14 @@ func RunTest(
 	if adapterInfo != nil {
 		adapterInfos = append(adapterInfos, adapterInfo)
 	}
-	if args, err = getServerArgs(scenario.Templates, adapterInfos, scenario.Configs); err != nil {
+	cfgs := scenario.Configs
+	if scenario.GetConfig != nil {
```

Comment:
can you please assert that there must be just one scenaros.Configs or scenario.GetConfig